specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2018-09-01/containerregistry_build.json

```diff
@@ -788,6 +783,11 @@
             "name": "ProvisioningState",
             "modelAsString": true
           }
+        },
```

Comment:
The order of fields can be important in some languages. For example, C#/Javas constructors could be generated according to the order in the swagger file.

Reply:
I reverted this change. SwashBuckle generated with a different order this time.